examples/client/encrypted_tagged_data_message.c

```diff
@@ -0,0 +1,122 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * @brief A simple example of sending an encrypted data to the Tangle.
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "client/api/restful/get_message.h"
+#include "client/api/restful/send_tagged_data.h"
+#include "core/models/payloads/tagged_data.h"
+
+#define XOR_KEY 'S'
+#define TAG "iota.c\xF0\x9F\xA6\x8B"
+#define DATA "Hello from encrypted data message example."
+
+/**
+ * @brief Simple encryption/decryption algorithm for this example
+ *
+ * @param src A string for encryption/decryption
+ * @param dest A buffer which stores an encrypted/decrypted data
+ * @param dest_len A buffer length
+ * @return int 0 on success
+ */
+static int xor_encrypt_decrypt(byte_t *src, size_t src_len, byte_t *dest, size_t dest_len) {
+  if (src_len > dest_len - 1) {
+    // destination buffer too small
+    return -1;
+  }
+
+  for (size_t i = 0; i < src_len; i++) {
+    dest[i] = src[i] ^ XOR_KEY;
+  }
+  dest[src_len] = '\0';
+
+  return 0;
+}
+
+int main(void) {
+  iota_client_conf_t ctx = {.host = "localhost", .port = 443, .use_tls = true};
```

Comment:
Should we change host to "chrysalis-nodes.iota.org" and later change to stardust public testnet once available?

Reply:
I'm not sure because anyway `chrysalis-nodes.iota.org` doesn't support new protocol and API changes.

Reply:
Thanks. It makes sense, let us keep localhost and later we can change to new tesnet.